src/Controller/ApiController.php

```diff
@@ -89,6 +75,10 @@ public function synchroAction(Request $request): \Symfony\Component\HttpFoundati
 
             // Get input data
             $data = json_decode($request->getContent(), true);
+            $force = true;
+            if ($data['rule'] == 'ALL') {
```

Comment:
**question(non-blocking)**:  Shouldn't we use === here instead of == ?

Reply:
Hi @EstelleMyddleware , I think ALL can be only a text, it isn't like False, empty and 0 that can return the same result with a ==. So no need to change this.